### PR TITLE
[MIG] web_timeline: Migration to 16.0

### PR DESCRIPTION
**Pull request by bvanjeelharia**

### CODE REVIEW

**Comment by chandni299** [comment]
Hello @bvanjeelharia, Made a functional review.
1. The timeline icon from `view_mode` is not displayed correctly if I try the below code as an example

2. We should improve the `README.rst` file in the given `.xml` example as we have fields with different names and actions in model `project.task`  like in `16.0`  and `15.0`version

```
<?xml version="1.0" encoding="utf-8"?>
<odoo>

    <record id="view_task_timeline" model="ir.ui.view">
        <field name="model">project.task</field>
        <field name="type">timeline</field>
        <field name="arch" type="xml">
            <timeline date_start="date_assign"
                date_stop="date_end"
                string="Tasks"
                default_group_by="user_ids"
                event_open_popup="true"
                zoomKey="ctrlKey"
                colors="#ec7063:user_ids == false;#2ecb71:kanban_state=='done';"
                dependency_arrow="dependent_ids">
                <field name="user_ids" />
                <templates>
                    <div t-name="timeline-item">
                        <div t-esc="record.display_name" /> Assigned to: <span
                            t-esc="record.user_ids[1]" />
                    </div>
                </templates>
            </timeline>
        </field>
    </record>

    <record id="project.action_view_all_task" model="ir.actions.act_window">
        <field name="view_mode">kanban,tree,form,calendar,pivot,graph,activity,timeline</field>
    </record>

</odoo>
```

**Comment by bvanjeelharia** [comment]
> 

Hello @chandni299 , I have made the changes as per your suggestions

File: web_timeline/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="utf-8"?>
+    <odoo>
+        <record id="view_task_timeline" model="ir.ui.view">
+            <field name="model">project.task</field>
+            <field name="type">timeline</field>
+            <field name="arch" type="xml">
+                <timeline date_start="date_start"
```
Comment (chandni299) [inline web_timeline/readme/CONFIGURE.rst]: We should change field to `date_assign` as we do not have field `date_start` in model `project.task`

**Review by chandni299 — APPROVED** [review]
Thank you @bvanjeelharia. We can see the Timeline icon now :)

**Comment by bvanjeelharia** [comment]
Can someone review and approve this?

**Review by tarteo — CHANGES_REQUESTED** [review]
I get an error when I double click a timeline item (with event_open_popup="true"):
```

UncaughtPromiseError > TypeError
Uncaught Promise > FormView is not a constructor

```

Apart from that everything seems to be working fine!

**Review by tarteo — APPROVED** [review]
It works now! Thanks!

**Pull request merged**